### ISSUE TITLE
URGENT Bad address on Error page

### ISSUE THREAD

**Issue by burlexpo** [bethlakshmi/GBE2#1162]
I tried to figure out if I can fix this and I don't think I can.  Our error page says that people can contact "webdev@burlexpo.com"  but that's not our domain.  It should be "webdev@burlesque-expo.com"

**Comment by bethlakshmi** [comment]
Can you tell me how to replicate the error?

**Comment by eclecticmagpie** [comment]
I've condensed my two notes to Scratch into one, and added some more detail.

Original Problem:
I got an error page when I responded to the request for volunteers to sign
up that you just sent out:

https://www.google.com/url?hl=en&q=http://../i-want/
volunteer-opps/&source=gmail&ust=1509768216144000&usg=
AFQjCNFPOHvKv7W7Sxyt1QmNPUS8jTRQnA

(note: this is what the URL was redirected to, not the actual URL on the
link, which appears to be "../i-want/volunteer-opps/"  This redirected to
the link above, but I don't know if that was a browser-dependent response;
I'm using Chrome on a Mac.

When I went directly to the website, and clicked on the volunteer form, I
got a different error, handled by _your_ error handler:

I started here (which looked normal):
http://burlesque-expo.com/i-want/volunteer-opps/

and wound up here (with an error displaying):
http://burlesque-expo.com/volunteer/bid/

At this point, the wrong address for bug reports was shown.
